index.js

```diff
@@ -2195,6 +2196,36 @@
         return filter(is(type), Just(x));
       });
 
+  //. ### Numeric
```

Comment:
I'd prefer __Number__, since the other titles—except __Classify__ which we could rename __Classification__—are nouns.